### PR TITLE
[web-wallet] PostMessage migration and production layout

### PR DESCRIPTION
**Pull request by kent-white**
## Motivation
Using postMessage:
runtime.sendMessage was great buttttttt we had to define what hosts could use the extension and also provide a extension id which would change in development. postMessage between the injected script and the contentscript does not require any permissions so it will be better for the upcoming hackathon. It's not super pretty though so I filed a task (#805) to make this better, there are a bunch of things I already tried that didn't work so I'll fill out that issue more when I have time.

Layout in production mode:
When we build the app for extension we don't want to have the "extension"/"fullscreen" options.

## Test Plan

Screen Shot 2022-05-04 at 8 45 53 PM

### CODE REVIEW

File: ecosystem/web-wallet/public/inpage.js [hunk 1]
```diff
@@ -1,31 +1,49 @@
...
+          }
         }
-        return true
       })
     })
   }
 
-  signTransaction (transaction, completion) {
+  signTransaction (transaction) {
+    const id = this.requestId++
```
Comment (hariria) [inline ecosystem/web-wallet/public/inpage.js:32]: Same as above

File: ecosystem/web-wallet/src/pages/Login.tsx [hunk 2]
```diff
@@ -83,8 +83,7 @@ function Login () {
             }
           </Box>
         </Center>
-        <Heading textAlign="center">Wallet</Heading>
-        <Text textAlign="center" pb={8} color={secondaryTextColor[colorMode]}>The official Aptos crypto wallet</Text>
```
Comment (hariria) [inline ecosystem/web-wallet/src/pages/Login.tsx:87]: Can we keep this but just have it say `An Aptos crypto wallet`. Looks weird without any subtext

**Review by hariria — APPROVED** [review]
Some nits, otherwise LGTM

File: ecosystem/web-wallet/public/inpage.js [hunk 3]
```diff
@@ -1,31 +1,49 @@
...
-
 class Web3 {
+  requestId;
+
+  constructor() {
+    this.requestId = 0
+  }
+
   account () {
+    const id = this.requestId++
```
[the remark this answers is not in the document]
Comment (kent-white) [reply ecosystem/web-wallet/public/inpage.js:12]: it is unlikely to happen but this.requestid can be incremented before getting the response back which kind of makes the the id useless

**Pull request merged**